src/storage/mvcc/reader/util.rs

```diff
@@ -15,12 +15,26 @@ use storage::mvcc::{Error, Result};
 use storage::mvcc::{Lock, LockType, Write};
 use storage::{Cursor, Iterator, Key, Statistics, Value};
 
-/// Checks whether the lock conflicts with the given `ts`. If there is no conflict,
-/// the safe `ts` will be returned.
-pub fn check_lock(key: &Key, ts: u64, lock: &Lock) -> Result<u64> {
+/// Representing check lock result.
+#[derive(Debug)]
+pub enum CheckLockResult {
+    /// Key is locked. The key lock error is included.
+    Locked(Error),
+
+    /// Key is not locked.
+    NotLocked,
```

Comment:
NotLocked or NoLock?

Reply:
NotLocked might be better? Because there could be a lock for the key, but the lock could be not valid for the given ts.